yo1k/tic_tac_toe/core/game.py

```diff
@@ -287,24 +290,24 @@ def __ready(state: State, player_idx: int):
             state.phase = Phase.INROUND
 
     @staticmethod
-    def __set_outround(state: State):
+    def __set_outround(state: State) -> None:
```

Comment:
Let's rename this method to `__end_round`?

Reply:
Done.